### ISSUE TITLE
Exporting html file of map using ipywidgets, and later trying to view map in browser

### ISSUE THREAD

**Issue by akanevs** [pbugnion/gmaps#175]
I am using the ipywidget's embed_minimal_html command to output the resulting html file of the google map image from jupyter notebook, which it definitely outputs. However, I don't understand why I am not able to see the map (using this html file) properly in a web browser afterwards. I get a blank page, and it seems to give the following error:

Uncaught ReferenceError: define is not defined
    at unpkg.com/@jupyter-widgets/html-manager@0.10.1/dist/embed-amd.js:1

Thank you,
Alex

**Comment by pbugnion** [comment]
Thanks for raising an issue. You are using ipywidgets 7.0.x, which was released a week ago. The stable version of `gmaps`, 0.5.4, is not compatible with 7.0.0. 

We are currently working on porting gmaps to ipywidgets 7. I've tested embedding specifically. 

There is a prerelease of gmaps out, 0.6.0-alpha6, which works with ipywidgets 7.

Install it with:

```
pip install gmaps==0.6.0-alpha6
jupyter nbextension enable --py --sys-prefix gmaps
```

Let us know how it goes!
